vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -237,6 +237,18 @@ unit:A-SEC
   rdfs:label "A SEC" ;
   skos:prefLabel "ampere second" ;
 .
+unit:A2-HR
```

Comment:
I don't understand this unit. The unit for measuring a Voltage Loss is the Volt.

Reply:
Please see the references I gave. I am glad that you take issue with these because I have no basis to question them; I just look them up and if I find references then I no longer question the unit but look for an understanding of its quantity kind.

One thing I found out is that some people confuse Voltage Loss with Voltage Drop and that they aren't the same.

Reply:
Recommend the "holding pen" for this.